### DISCUSSION TITLE
Acmebot v4 for .NET 8

### DISCUSSION THREAD

**Discussion by shibayan** [polymind-inc/acmebot#720] [Announcements]
Microsoft has released In-Proc .NET 8 support for Azure Functions. .NET 6 to 8 migration is highly compatible, so you can run Acmebot v4 on .NET 8 LTS by simply executing the following Azure CLI command.

```
az functionapp config set -g <groupName> -n <appName> --net-framework-version v8.0
az functionapp config appsettings set -g <groupName> -n <appName> --settings FUNCTIONS_INPROC_NET8_ENABLED=1
```

If the runtime version you can check from the Azure Portal is `4.8xx.x.xxxxx`, it is running on .NET 8.

**Comment by saqib-s** [comment]
I've made this change, and it removed the function app host not running error, and the portal loads but I still have the "External startup exception" present in the notifications. I have confirmed that the two changes above were made successfully.  Am I missing something? 

<img width="1395" height="255" alt="image" src="https://github.com/user-attachments/assets/73766c2a-3509-44ef-9149-511dc4de2444" />
<img width="853" height="158" alt="image" src="https://github.com/user-attachments/assets/75f8f0d1-cdff-4c77-a989-fd2afb9f9be8" />

<img width="625" height="193" alt="image" src="https://github.com/user-attachments/assets/d1f9ba3d-b242-46dc-92c0-64eba6470ac7" />

**Reply by shibayan** [reply]
Please check the last occurrence date and time. Since the error is no longer occurring, the warning should stop appearing after a while.

**Reply by saqib-s** [reply]
Thanks again, seems like the last occurrence was just before making the update, appreciate the quick response!

**Comment by StandBackBurrito** [comment]
I made these updates, but I can not get the function to start. I get the error 

```
Microsoft.Azure.WebJobs.Script: Error building configuration in an external startup class. System.Private.CoreLib: Could not load file or assembly 'KeyVault.Acmebot, Version=4.3.1.0, Culture=neutral, PublicKeyToken=null'.
```
<img width="1575" height="174" alt="image" src="https://github.com/user-attachments/assets/f0f19d0b-65a8-405a-84e4-8ceb4a44101a" />

Is there something else I need to change?

**Reply by shibayan** [reply]
You shouldn't need any settings beyond what's written here, but please access the Azure Portal and double-check that the settings are correct.

**Reply by StandBackBurrito** [reply]
Thanks, I was not able to find anything. I was able to re-deploy and get it working again
